### PR TITLE
Infer inputs from the current dir when none given

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@davidwengier FYI

File: src/mdv/Arguments.cs [hunk 1]
```diff
@@ -113,5 +127,37 @@ private static bool ParseFlagArg(string[] args, string name, bool defaultValue)
...
+            return false;
+        }
+
+        var deltasBuilder = ImmutableArray.CreateBuilder<(string, string)>();
+        var i = 1;
+        while (true)
+        {
+            var pathWithoutExtension = System.IO.Path.Combine(dir, i.ToString());
+            var ilPath = pathWithoutExtension + ".il";
+            var mdPath = pathWithoutExtension + ".md";
```
Comment (davidwengier) [inline src/mdv/Arguments.cs:147]: Before this gets encased in code forever, would it be better to use `.metadata`? 8.3 filenames are no longer a thing, and `.md` are pretty well known as markdown files.

I know its not an issue here, and its unlikely anyone would think to run mdv on a random markdown file, but thought I'd mention it.

**Review by davidwengier — APPROVED** [review]
Next step is to have Roslyn automatically call mdv after it writes out the files, right? :D

File: src/mdv/Arguments.cs (hunk 1 above)
Comment (tmat) [reply src/mdv/Arguments.cs:147]: Hmm, good point but our ENC logger already uses ".md". I can change that though.

**Pull request merged**